CDPEntitiesApi.ts

```diff
@@ -42,11 +35,11 @@ export type CDPEntitiesApi = {
     workspaces: EntityApi<CDPEntityDef<Workspace>, {
         applibrary: EntityApi<CDPEntityDef<Connector>>,
         global: EntityApi<never, {
-            applibrary: EntityApi<CDPEntityDef<Connector>>,
+            applibrary: EntityApi<CDPEntityDef<Connector[]>>,
```

Comment:
why? the type provided to `CDPEntityDev` represents the response of a single GET